Manager/CurrencyManager.php

```diff
@@ -40,4 +49,29 @@ public function getAll()
     {
         return $this->em->getRepository($this->currencyClassName)->findAll();
     }
+
+    /**
+     * @return int - Count of create currencies
+     *
+     * @throws ORMInvalidArgumentException
+     * @throws OptimisticLockException
+     */
+    public function createBaseCurrencies()
```

Comment:
> Please add functionality for adding Currencies through the CurrencyManager
I actually meant by this following: 
* add CurrencyManager::addCurrency($code) which will cover EntityManager calls
* add ICurrencyManager::addCurrency($code)

And most of this code will stay in the Command, it's not necessary for CurrencyManager to know about some base currencies